<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowConfigItem</class>
 <widget class="QWidget" name="WindowConfigItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>68</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>935</width>
    <height>68</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxName">
     <property name="title">
      <string>Имя</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditWindowName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Action Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxColor">
     <property name="title">
      <string>Цвет</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>#ff000000</string>
        </property>
        <property name="maxLength">
         <number>9</number>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSelectColor">
        <property name="text">
         <string>Выбрать цвет</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPosition">
     <property name="title">
      <string>Координаты</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelPositionX">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxPositionX">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelPositionY">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxPositionY">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSize">
     <property name="title">
      <string>Размер</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelWidth">
        <property name="text">
         <string>Ш:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelHeight">
        <property name="text">
         <string>В:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxHeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>150</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAction">
     <property name="title">
      <string>Дейтсвие</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelActionType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Тип:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxActionType">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonShow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Показать</string>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDelete">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/icons/remove.png</normaloff>:/icons/icons/remove.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonDelete</sender>
   <signal>pressed()</signal>
   <receiver>WindowConfigItem</receiver>
   <slot>onPushButtonDeletePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>931</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>713</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSelectColor</sender>
   <signal>pressed()</signal>
   <receiver>WindowConfigItem</receiver>
   <slot>onPushButtonSelectColorPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonShow</sender>
   <signal>pressed()</signal>
   <receiver>WindowConfigItem</receiver>
   <slot>onPushButtonShowPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>931</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>713</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxPositionX</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WindowConfigItem</receiver>
   <slot>onXChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxPositionY</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WindowConfigItem</receiver>
   <slot>onYChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxWidth</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WindowConfigItem</receiver>
   <slot>onWidthChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxHeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WindowConfigItem</receiver>
   <slot>onHeightChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>601</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxActionType</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>WindowConfigItem</receiver>
   <slot>onComboBoxActionTypeCurrentTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPushButtonDeletePressed()</slot>
  <slot>onPushButtonSelectColorPressed()</slot>
  <slot>onPushButtonShowPressed()</slot>
  <slot>onXChanged(int)</slot>
  <slot>onYChanged(int)</slot>
  <slot>onWidthChanged(int)</slot>
  <slot>onHeightChanged(int)</slot>
  <slot>onComboBoxActionTypeCurrentTextChanged(QString)</slot>
 </slots>
</ui>
